docarray/array/storage/weaviate/backend.py

```diff
@@ -0,0 +1,290 @@
+from dataclasses import dataclass, field
+import uuid
+import itertools
+from typing import (
+    Generator,
+    Iterator,
+    Dict,
+    Sequence,
+    Optional,
+    TYPE_CHECKING,
+    Union,
+    Tuple,
+    List,
+)
+
+import weaviate
```

Comment:
move the `import weaviate` under `_init_storage()` to avoid the dependency missing problem.

Reply:
Will do, thanks.